ctags/main/main.c

```diff
@@ -46,7 +46,7 @@
 #  include <sys/types.h>  /* required by dirent.h */
 # endif
 # include <dirent.h>  /* to declare opendir() */
-# undef boolean
+# undef bool
```

Comment:
@techee I already reviewed the whole diff, so while I might of course have missed something, I flagged what I saw, and the rest seemed just fine.